packages/components/src/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState, ReactNode } from 'react';
+import React from 'react';
+import { type DropdownProps } from "@/dropdown/Dropdown.types";
+import { globalStyles } from "@/index.styled";
+import { DropdownContainer, DropdownLabel, DropdownChildren, DropdownItem, darkTheme } from "@/dropdown/Dropdown.styled"; 
+import chevronDown from "@/assets/chevron-down.svg"
+
+export const Dropdown = ({
+        label = "Select...", 
+        theme, 
+        showingOptions: initialShowingOptions, 
```

Comment:
Make the 'showingOptions' prop optional. Another option is to rename it to something more simple e.g 'expand'